practice.txt

```diff
@@ -0,0 +1,2 @@
+This is my practice portfolio design.
```

Comment:
explain it! xd 💕

Reply:
nothing. just added a line so that everyone know why its created.